Sming/SmingCore/Network/MqttClient.cpp

```diff
@@ -77,6 +77,10 @@ bool MqttClient::setWill(const String& topic, const String& message, int QoS, bo
 bool MqttClient::connect(const URL& url, const String& clientName, uint32_t sslOptions)
 {
 	this->url = url;
+	if (not (url.Protocol == "mqtt" || url.Protocol == "mqtts")) {
+		debug_e("Invalid protocol");
```

Comment:
@frankdownunder Maybe the error message should be a bit more descriptive and explain that only mqtt and mqtts protocols are allowed.